examples/src/main/java/nl/knaw/dans/lib/dataverse/smoketests/README.md

```diff
@@ -22,4 +22,6 @@ TODO:
 * maybe various methods of other APIs
 * rewrite tests such that we won't need a smoke test configuration at all, such as:
   * create user with a UUID in its name
-  * create a dataverse with a UUID in its name
\ No newline at end of file
+  * create a dataverse with a UUID in its name
+* Setup that (temporarily) makes DANS specific metadata blocks optional
+  for DataverseCreateDataset and DataverseImportDataset examples, introduced in PR #60.
```

Comment:
While providing this custom metadata, I missed builders for the metadata. Would it be an idea to add builders to the client lib? It could make other code less verbose too.